src/components/account/account.service.js

```diff
@@ -60,6 +60,36 @@ module.exports.getByEmail = async (email) => {
     }
 };
 
+/**
+ * Changes the user's password if the current password provided is valid
+ * @throws {Unauthorized} If the current password is invalid.
+ * @throws {BadRequest} If the required passwords are not provided.
+ * @param {String} userId User identifier
+ * @param {*} passwords Current and new password.
+ */
```

Comment:
Your code works and its very clear!

the only thing i noticed its that maybe you could add the `InternalServerError` exception to the @throws options.

Or maybe its not necessary. You know whats best!

Reply:
nice catch!